### PR TITLE
fix(public-debt-payment-plan): Info screen for users that have no debts

### PR DESCRIPTION
**Pull request by bensijons**
# Info screen for users that have no debts

https://app.clickup.com/t/1mugzfj

## What

Info screen for users that have no debts, so that users can't continue with the application.

## Why

Requested after testing

## Screenshots / Gifs

image

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/public-debt-payment-plan/src/lib/paymentPlanUtils.ts [hunk 1]
```diff
@@ -14,6 +15,11 @@ export const prerequisitesFailed = (data: ExternalData) => {
     | PaymentScheduleConditions
     | undefined
 
+    const debts = (data as PaymentPlanExternalData)
+    .paymentPlanPrerequisites?.data?.debts as PaymentScheduleDebts[]
+
+    console.log('debts', debts)
```
Comment (sigruntg) [inline libs/application/templates/public-debt-payment-plan/src/lib/paymentPlanUtils.ts:21]: console.log

**Review by sigruntg — CHANGES_REQUESTED** [review]
Looks great, just remove the console.log

File: libs/application/templates/public-debt-payment-plan/src/lib/paymentPlanUtils.ts (hunk 1 above)
Comment (Thorrik58) [inline libs/application/templates/public-debt-payment-plan/src/lib/paymentPlanUtils.ts:21]: Console.log
```suggestion
```

**Review by Thorrik58 — CHANGES_REQUESTED** [review]
LGTM apart from the console log. There is a small thing that bugs me and that is that the application will stay listed when the prerequisites fail (that you have a debt) and we might want to keep the application unlisted until that prerequisite passes. 
We are doing this with the health insurance application but it is a bigger feature if we want to go that route. Just a thought.

**Review by sigruntg — APPROVED** [review]

**Review by Thorrik58 — APPROVED** [review]

**Pull request merged**